docs/appendices/backend-writers-guide.rst

```diff
@@ -771,6 +813,147 @@ other update/remove functionality at a later stage.
   must be added to the zone. ``rrset`` can be empty in which case the
   method is used to remove a RRset.
 
+Domain metadata support
+-----------------------
+
+As described in :ref:`per-zone-settings-domain-metadata`, each served zone can have “metadata”. Such metadata determines how this zone behaves in certain circumstances.
```

Comment:
unsure if the quoting around metadata is the correct style.

Reply:
The idea was to link to https://docs.powerdns.com/authoritative/domainmetadata.html#per-zone-settings-domain-metadata but I'm afraid it's not working, I'll have a look.